src/query/atomWithInfiniteQuery.ts

```diff
@@ -192,8 +233,8 @@ export function atomWithInfiniteQuery<
       const { observer } = get(queryDataAtom)
```

Comment:
On second thought, as [useInfiniteQuery](https://react-query.tanstack.com/reference/useInfiniteQuery) doesn't support customizing options anyway. Maybe, we don't need to specify it.
How about using the same `options`?
```suggestion
      const { observer, options } = get(queryDataAtom)
```

Or, we don't even need to pass `options` to `overserver.refetch()`?

Reply:
I'm not very similar with this `useInfiniteQuery`, u could deal this with ur  thought